dist/adf-widget-scm.js

```diff
@@ -183,17 +183,22 @@ angular.module('adf.widget.scm', ['adf.provider', 'chart.js', 'ngSanitize', 'btf
+$templateCache.put("{widgetsPath}/scm/src/markdownPreview/view.html","<style>\n  div.markdownContent{\n    overflow: auto;\n    width: 100%;\n  }\n</style><div class=\"alert alert-info\" ng-if=\"vm.fileContent === null\">Please configure a specific file</div><div ng-if=vm.fileContent btf-markdown=vm.fileContent class=markdownContent></div><div class=\"alert alert-danger\" ng-if=\"vm.fileContent.status == 500 || vm.fileContent.status == 404\"><b>Error {{vm.fileContent.status}}</b> The Markdown file was not found. This may be because the file does not exist or the SCM Manager is not available at the moment.</div>");}]);
 
 
 angular.module('adf.widget.scm')
-  .controller('MarkdownPreviewEditController', ["repositories", "SCM", function (repositories, SCM) {
+  .controller('MarkdownPreviewEditController', ["repositories", "SCM", "$scope", "$sce", function (repositories, SCM, $scope, $sce) {
     var vm = this;
     vm.repositories = repositories;
 
+    // tooltips
+    $scope.repositoryTooltip = $sce.trustAsHtml('Select the repository which stores the desired Markdown.');
+    $scope.branchTooltip = $sce.trustAsHtml('Select the branch on which the desired Markdown file can be found.');
+    $scope.pathToMarkdownTooltip = $sce.trustAsHtml('The path where the Markdown file can be found within the repository (e.g. <b>docs/README.md</b>). Note that the path is case sensitive. For files in the root dir of the repository add <b>NO</b> leading slash.');
```

Comment:
“Enter the path to the Markdown file with a leading slash. The path is case sensitive. Files in the root dir start <b>without</b> a leading slash.